### PR TITLE
Fix entities teleporting to the wrong world

### PR DESCRIPTION
**Pull request by Gaider10**
Fixes entities trying to teleport to normal worlds when going through portals

### CODE REVIEW

File: src/main/java/com/iridium/iridiumskyblock/listeners/EntityPortalListener.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+public class EntityPortalListener implements Listener {
+
+    @EventHandler
+    public void onEntityPortal(EntityPortalEvent event) {
+        if(event.getTo() != null && event.getTo().getWorld() != null) {
+            IslandManager islandManager = IridiumSkyblock.getInstance().getIslandManager();
+            IridiumSkyblock.getInstance().getIslandManager().getIslandViaLocation(event.getFrom()).ifPresent(island -> {
+                World worldFrom = event.getFrom().getWorld();
+                String worldToName = event.getTo().getWorld().getName();
+                if ((worldToName.equals("world_nether") && Objects.equals(worldFrom, islandManager.getWorld())) || (worldToName.equals("world") && Objects.equals(worldFrom, islandManager.getNetherWorld()))) {
```
Comment (MertUnverdi) [inline src/main/java/com/iridium/iridiumskyblock/listeners/EntityPortalListener.java:22]: Every world name is "world"?
Make it via Bukkit#getWorld

Comment (Gaider10) [reply src/main/java/com/iridium/iridiumskyblock/listeners/EntityPortalListener.java:22]: I'm pretty sure that there can't be multiple worlds with the same name

Comment (dlsf) [reply src/main/java/com/iridium/iridiumskyblock/listeners/EntityPortalListener.java:22]: I guess he meant that users may not use these hardcoded world names

**Comment by MertUnverdi** [comment]
stupid peaches.

**Pull request closed without merging**